### PR TITLE
Add option to avoid triggering the loclist on errors

### PR DESCRIPTION
**Pull request by sigzegv**
Usefull when you already use another diagnotics system.

### CODE REVIEW

File: autoload/goimports.vim [hunk 1]
```diff
@@ -10,6 +10,11 @@ if exists('g:goimports_simplify_cmd')
   let s:goimports_simplify_cmd = g:goimports_simplify_cmd
 endif
 
+let s:goimports_show_loclist = 1
+if exists('g:goimports_show_loclist')
+  let s:goimports_show_loclist = g:goimports_show_loclist
+endif
+
```
Comment (mattn) [inline autoload/goimports.vim]: Thank you. Please remove those lines.

File: autoload/goimports.vim [hunk 2]
```diff
@@ -89,7 +94,7 @@ function! goimports#Run() abort
 
   for f in l:filters
     let [l:out, l:err] = function(f)()
-    if l:err != 0
+    if l:err != 0 && s:goimports_show_loclist == 1
```
Comment (mattn) [inline autoload/goimports.vim]:
```vim
if l:err != 0 && get(g:, 'goimports_show_loclist', 1)
```

File: autoload/goimports.vim (hunk 1 above)
Comment (sigzegv) [reply autoload/goimports.vim]: done, I juste thought it was your coding style to prepare vars (like import_cmd vars)

**Comment by mattn** [comment]
Thank you

**Pull request merged**